android/crt/build.gradle

```diff
@@ -115,18 +116,21 @@ android {
     compileOptions {
         sourceCompatibility = 1.8
         targetCompatibility = 1.8
+        coreLibraryDesugaringEnabled true
```

Comment:
Can you put a comment here on what this does, or link to docs? I need to know EVERYTHING about whatever "desugaring" is

Reply:
Added a comment. It allows targeting a subset of Java 8 API's on older versions of Android. The Android tooling will add a backport library behind the scenes if necessary. For a library this really just gets around lint issues since the desugaring should happen when building an APK not an aar. 

See:
* https://jakewharton.com/d8-library-desugaring/
* https://developer.android.com/studio/write/java8-support#library-desugaring
* https://developer.android.com/studio/write/java8-support-table